templates/rc.erb

```diff
@@ -0,0 +1,263 @@
+#! /bin/sh
+#
+# rc
+#
+# Starts/stops services on runlevel changes.
+#
+# Optimization: A start script is not run when the service was already
+# configured to run in the previous runlevel.  A stop script is not run
+# when the the service was already configured not to run in the previous
+# runlevel.
+#
+# Authors:
+# 	Miquel van Smoorenburg <miquels@cistron.nl>
+# 	Bruce Perens <Bruce@Pixar.com>
```

Comment:
@artem-sidorenko Again.. As I am super new to puppet and hardening, just started last month. This is just a template erb file. which I took it from my debian system. Was just experimenting with implementing rc.